Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -0,0 +1,415 @@
+        ],
+        2 : [
+            'Geometry/MuonCommonData/data/muonNumbering/2017/v2/muonNumbering.xml',
+        ],
+        3 : [
+            'Geometry/MuonSimData/data/v2/muonSens.xml',
+            'Geometry/DTGeometryBuilder/data/dtSpecsFilter/2021/v1/dtSpecsFilter.xml',
+            'Geometry/CSCGeometryBuilder/data/cscSpecsFilter.xml',
+            'Geometry/CSCGeometryBuilder/data/cscSpecs.xml',
+            'Geometry/RPCGeometryBuilder/data/RPCSpecs.xml',
+            'Geometry/GEMGeometryBuilder/data/GEMSpecsFilter17.xml',
+            'Geometry/GEMGeometryBuilder/data/v4/GEMSpecs.xml',            
+        ],
+        4 : [
+            'Geometry/MuonSimData/data/muonProdCuts/2021/v1/muonProdCuts.xml',
+        ],
+        "sim" : [
+            'from Geometry.MuonNumbering.muonGeometryConstants_cff import *',
+        ],
+        "reco" : [
+            'from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *',
+            'from RecoMuon.DetLayers.muonDetLayerGeometry_cfi import *',
+            'from Geometry.GEMGeometryBuilder.gemGeometry_cfi import *',
+            'from Geometry.CSCGeometryBuilder.idealForDigiCscGeometry_cff import *',
+            'from Geometry.DTGeometryBuilder.idealForDigiDtGeometry_cff import *',
+        ],
+        "era" : "phase2_muon, run3_GEM",
+    },
+}
+
+forwardDict = {
+    "abbrev" : "F",
+    "name" : "forward",
+    "default" : 1,
+    "F1" : {
+        1 : [
+            'Geometry/ForwardCommonData/data/forward/2021/v1/forward.xml',
+            'Geometry/ForwardCommonData/data/totemt2/2021/v1/totemt2.xml',
+            'Geometry/ForwardCommonData/data/forwardshield/2021/v1/forwardshield.xml',
+            'Geometry/ForwardCommonData/data/brmrotations.xml',
+            'Geometry/ForwardCommonData/data/brm/2021/v1/brm.xml',
+            'Geometry/ForwardCommonData/data/zdcmaterials.xml',
+            'Geometry/ForwardCommonData/data/lumimaterials.xml',
+            'Geometry/ForwardCommonData/data/zdcrotations.xml',
+            'Geometry/ForwardCommonData/data/lumirotations.xml',
+            'Geometry/ForwardCommonData/data/zdc/2021/v1/zdc.xml',
+            'Geometry/ForwardCommonData/data/zdclumi/2021/v1/zdclumi.xml',
+            'Geometry/ForwardCommonData/data/cmszdc.xml',
+        ],
+        3 : [
+            'Geometry/ForwardCommonData/data/brmsens.xml',
+            'Geometry/ForwardSimData/data/totemsensT2/2021/totemsensT2.xml',
+            'Geometry/ForwardSimData/data/zdcsens.xml',
+        ],
+        4 : [
+            'Geometry/ForwardSimData/data/zdcProdCuts.xml',
+            'Geometry/ForwardSimData/data/ForwardShieldProdCuts.xml',
+        ],
+        "reco" :[
+            'from Geometry.ForwardGeometry.ForwardGeometry_cfi import *',
+        ]
+    }
+}
+
+ctppsDict = {
+    "abbrev" : "P",
+    "name" : "ctpps",
+    "default" : 1,
+    "P1" : {
+        1 : [
+            'Geometry/VeryForwardData/data/pps/2021/v1/pps.xml',
```

Comment:
@cvuosalo - this is the section for Luis to fill in.

Reply:
Hi.. I 'm a bit confused, where should I put the pps geometry files? I thoough I would work on the Geometry/CMSCommon/python/cmsExtendedGeometry2021XML_cfi.py
So, where should I put the XML files? Or should I put in both files?
Another question is that as I told in the other PR, I could not make the single pps.xml file to work, only when including all of them in the cmsExtendedGeometry2021XML_cfi.py. Should I test again using the dictionary above?
Thanks.

Reply:
@mundim - as soon as this PR is merged you will not be able to modify the `cmsExtendedGeometry2021XML_cfi.py` by hand. Instead, you'd have to put all the xml files in this dictionary replacing the line 395, then generating the `cmsExtendedGeometry2021XML_cfi.py` by
```
python ./Configuration/Geometry/scripts/generate2021Geometry.py
```